packages/helper-image/src/image-helper.browser.js

```diff
@@ -0,0 +1,145 @@
+/**!
+*
```

Comment:
Just put this in index.js. Since there's only one implementation, we don't need the extra indirection.